components/badges/badges.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Badges</title>
+    <link rel="stylesheet" href="badges.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+  
+       <!-- 
+        BADGES
+     -->
+    <div class="box">
+        <div class="badges">
+        <div class="text__badge">
+            <span>type text here</span>
+            <span class="badge">25</span>
+        </div>
+        <div class="text__badge1">
+            <span>type text here</span>
+            <span class="badge1">2</span>
+        </div>
+        <div class="text__badge2">
+            <button>Button</button>
+            <span class="badge2">2</span>
+        </div>
+        <button type="button" class="text__badge3">
```

Comment:
naming can be improved here, badge1/2/3 does not give any info about the style/type

Reply:
ok done